File: Code/C#/HVPSAPI/Responses/GreetingResponse.cs
using Core.Messages.Messages;
using HVPSAPI.DataMemberNames.Messages;
using HVPSAPI.DataMemberNames.Responses;
using HVPSAPI.Messages;
using MessageTypes.Attributes;
using System.Runtime.Serialization;
using System.Text.Json.Serialization;
namespace HVPSAPI.Responses
{
    [DataMemberNamesClass(typeof(GreetingResponseDataMemberNames))]
    public class GreetingResponse
    {
        [JsonPropertyName(GreetingResponseDataMemberNames.LastAbortMessage)]
        [JsonInclude]
        [DataMember(Name = GreetingResponseDataMemberNames.LastAbortMessage)]
        public LastAbortMessage? LastAbortMessage { get; set; }
        [JsonPropertyName(GreetingResponseDataMemberNames.CoreDumpSummaryMessage)]
        [JsonInclude]
        [DataMember(Name = GreetingResponseDataMemberNames.CoreDumpSummaryMessage)]
        public CoreDumpSummaryMessage? CoreDumpSummaryMessage { get; set; }
    }
}

File: Code/JavaScript/client/src/messages/GreetingResponse.js
import LastAbortMessage from './LastAbortMessage';
import CoreDumpSummaryMessage from './CoreDumpSummaryMessage';
export default class GreetingResponse
{
 static fromJSON(o){
    const r = {};
    const n = (v)=>v!==undefined&&v!==null;
    let v;
    v = o["la"];
    if(n(v))
        r.lastAbortMessage=LastAbortMessage.fromJSON(v)
    v = o["cd"];
    if(n(v))
        r.coreDumpSummaryMessage=CoreDumpSummaryMessage.fromJSON(v)
   return r;
 }
}